test/api/requests/site.test.js

```diff
@@ -684,7 +684,44 @@ describe('Site API', () => {
           .expect(400)
       ).then((response) => {
         validateAgainstJSONSchema('DELETE', path, 400, response.body);
-        expect(response.body.message).to.equal('A site must have at least one user');
+        expect(response.body.message).to.equal('A site must have at least one user. If you want to remove the last user, delete the site from Settings -> Advanced.');
+        done();
+      })
+      .catch(done);
+    });
+
+    it('should respond with a 400 when the site owner attempts to delete the site', (done) => {
```

Comment:
this wording seems a little off. Maybe should be `should respond with a 400 when the site owner attempts to remove themself`

Reply:
You're right, I'm not sure what I was thinking here :D